### PR TITLE
feat: Add `RadioGroup` story

### PR DESCRIPTION
**Pull request by karelianpie**
## Motivation and Context

Add all our current components into Storybook via stories.

## Description

Implements a story for the `RadioGroup.tsx` component.

## Test and Deployment Plan

Ran storybook locally and observed the story, refer to the screenshot below:

## Screenshots (if appropriate):

Screenshot 2022-10-29 at 10 55 15

## Reviewers

@crabsinger 

## Related Issue

https://github.com/coordinape/coordinape/issues/1570

### CODE REVIEW

File: src/ui/RadioGroup/RadioGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  RadioGroupIndicator,
+  RadioGroupRadio,
+  RadioGroupRoot as RadioGroupRootComponent,
+} from './RadioGroup';
+
+export default {
+  title: 'Design System/Components/RadioGroup',
+  component: RadioGroupRootComponent,
+  decorators: [withDesign, Story => <Story />],
+  argTypes: {
```
Comment (crabsinger) [inline src/ui/RadioGroup/RadioGroup.stories.tsx:16]: this is nice... to remove all the noisy options around props that are not specific... is there a way to use shared argTypes across many components? Or maybe have a helper function for hiding these?

Comment (crabsinger) [reply src/ui/RadioGroup/RadioGroup.stories.tsx:16]: https://storybook.js.org/docs/react/api/argtypes#global-argtypes

Comment (crabsinger) [reply src/ui/RadioGroup/RadioGroup.stories.tsx:16]: Tried to move this to `preview` but didn't seem to work...

File: src/ui/RadioGroup/RadioGroup.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import { Flex, FormLabel } from 'ui';
+
+import {
+  RadioGroupIndicator,
+  RadioGroupRadio,
+  RadioGroupRoot as RadioGroupRootComponent,
+} from './RadioGroup';
+
+export default {
+  title: 'Design System/Components/RadioGroup',
```
[the remark this answers is not in the document]
Comment (karelianpie) [reply src/ui/RadioGroup/RadioGroup.stories.tsx]: I ran your PR locally and was getting one level too deep with the inferred titles, i.e.:

```
- Panel
  - Panel
    - Panel
      - Story A
      - Story B
      - Story C
```

Unsure if that can be resolved, for now I've removed the title with https://github.com/coordinape/coordinape/pull/1594/commits/ebd0c4b212b610fa526125adfc22bd66d96504a4

File: src/ui/RadioGroup/RadioGroup.stories.tsx (hunk 1 above)
Comment (karelianpie) [reply src/ui/RadioGroup/RadioGroup.stories.tsx]: It work for me, perhaps you need to restart storybook, done with https://github.com/coordinape/coordinape/pull/1594/commits/ee28e494e13e1eec958aeb4acec2b91d091f2710

File: .storybook/preview.js [hunk 3]
```diff
@@ -21,3 +21,21 @@ export const parameters = {
     },
   },
 };
+
+export const argTypes = {
```
Comment (crabsinger) [inline .storybook/preview.js:25]: nice!

File: src/ui/RadioGroup/RadioGroup.stories.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (crabsinger) [reply src/ui/RadioGroup/RadioGroup.stories.tsx]: Hmm ...

Here's what I'm seeing:

image

@karelianpie

**Review by crabsinger — DISMISSED** [review]
Okay - approving - this looks great as a model for these stories @karelianpie.

File: src/ui/RadioGroup/RadioGroup.stories.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (karelianpie) [reply src/ui/RadioGroup/RadioGroup.stories.tsx]: This looks good, here's what I see is with Storybook `6.4.19`:

Menubar_and_Notification_Center

@crabsinger

**Review by wingfeatherwave — APPROVED** [review]

**Pull request merged**